### DISCUSSION TITLE
Sync movies marked as watched in Watchlist/Disover section

### DISCUSSION THREAD

**Discussion by danhew** [Taxel/PlexTraktSync#1420] [Q&A]
I've noticed that ptsync will now sync TV episodes I've [manually marked as watched](https://imgur.com/a/zqsBlj1) in the discover/watchlist sections - episodes that I haven't downloaded locally, but watched on a streaming service - which is great! But it only seems to work on TV episodes, and not movies.

I'm wondering if this behaviour is expected, or if anyone has gotten it to sync movies?

### UPDATE
I'm an idiot, and it turns out that I _did_ have those episodes locally downloaded after all. That explains why they were being synced.
This has now become a feature request, I guess

**Comment by bulletproof2k** [comment]
I have already asked a similar question and I hope that it is now in development :)

**Reply by danhew** [reply]
I was mistaken and the episodes that synced were in my Plex library. That explains why ptsync was seeing them as watched. 

Hopefully this is something they're considering adding!

**Reply by simonc56** [reply]
The feature request is #1142
